sorts/cyclic_sort.py

```diff
@@ -0,0 +1,52 @@
+"""
+This is a pure Python implementation of the Cyclic Sort algorithm.
+
+For doctests run following command:
+python -m doctest -v cyclic_sort.py
+or
+python3 -m doctest -v cyclic_sort.py
+For manual testing run:
+python cyclic_sort.py
+"""
+
+
+def cyclic_sort(nums: list) -> list:
```

Comment:
The algorithm is specified to sort integers, so a more correct typehint would be `list[int]` for the `nums` and the method output.